Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -838,18 +915,7 @@ bool JitArm64::DoJit(u32 em_address, JitBlock* b, u32 nextPC)
   js.numFloatingPointInst = 0;
 
   u8* const start = GetWritableCodePtr();
```

Comment:
Is there still much use for the temporary start variable? You could just set normalEntry directly and use normalEntry in the codeSize computation later, like in CachedInterpreter.cpp.

Reply:
Done.